### PR TITLE
Sheet and card modals

### PR DESCRIPTION
**Pull request by gewfy**
# Card sheets
| Add Session | Join Session | Session not found |
|-|-|-|
| image | image | Screenshot 2022-11-17 at 17 13 38 |

# Bottom sheets
| Session Details | Create Session Expanded | Request Host | 
|-|-|-|
| image | image | image |

### CODE REVIEW

File: client/src/lib/navigation/constants/routes.ts [hunk 1]
```diff
@@ -13,17 +13,18 @@ export type SessionStackProps = {
   OutroPortal: undefined;
 };
 
+export type AppStackProps = {
+  KillSwitch: undefined;
+  Tabs: NavigatorScreenParams<TabNavigatorProps>;
+  SessionStack: NavigatorScreenParams<SessionStackProps>;
+};
+
 export type ModalStackProps = {
```
Comment (swemail) [inline client/src/lib/navigation/constants/routes.ts:22]: Should `ModalStackProps` be the name of our main Stack? It's like `App` is a modal. Isn't `RootStackProps` for this better?

**Review by swemail — APPROVED** [review]
Work of art, as always!

File: client/src/lib/navigation/constants/routes.ts (hunk 1 above)
Comment (gewfy) [reply client/src/lib/navigation/constants/routes.ts:22]: I pondered the same. But it doesn't make sense when using these types to navigate later. I mean you navigate in the modal stack, not the root stack? In this case App is actually not a modal. The first child of the Modal Stack is what's being rendered behind the modals basically. Ehm, did that make sense? :D

Comment (swemail) [reply client/src/lib/navigation/constants/routes.ts:22]: It makes sort of sense 😂 Let's have it as this and feel it in action

**Pull request merged**